README.md

```diff
@@ -155,6 +157,43 @@ A helpful article on setting up WSL to work flawlesly with Desktop Docker for Wi
         ```
         export DOCKER_HOST=tcp://localhost:2375
         ```
+## Linked Tutorials
+
+### Create a local setup with Hugo and Docforge executables
+
+1. Download Hugo from [GitHub](https://github.com/gohugoio/hugo/releases).
+> **Note:** You need to download the extended version.
+2. Download Docforge from [GitHub](https://github.com/gardener/docforge/releases).
+> **Note:** In case of a problem with the executable, here is how to [build your local version of Docforge](https://github.com/gardener/website-generator#build-a-local-docforge-executable).
+2. Navigate to Local Disk C.
+3. Create a folder there and name it appropriately. 
+4. Extract the downloaded archive files into the folder.
+5. _(Optional)_ Delete the LICENSE and README files.
+
+### Build a local Docforge executable
+
+1. [Clone](https://github.com/gardener/website-generator#clone-a-repository) the [Docforge repository](https://github.com/gardener/docforge).
+2. Open a new terminal.
+3. In the terminal, navigate to the folder where you cloned Docforge.
+4. Enter `export LOCAL_BUILD=1 && ./.ci/build`.
+5. In your system, open the folder where you cloned Docforge.
+6. Open the "bin" folder and copy the "docforge" file there.
+7. Paste the file into the folder containing the Hugo executable and 
+8. **Optional:** Rename the file to "docforge.exe".
```

Comment:
mention that this step is only for Windows. Also the binary should be added in the PATH environment